src/examples/algorithms/HelloSP_TP.cpp

```diff
@@ -1,96 +0,0 @@
-/* ---------------------------------------------------------------------
```

Comment:
The Hello* code examples have typically used a very small amount of artificial data and printed out extensive information to show exactly how the algorithms work. The hotgym examples are intended more to show an end-to-end implementation for a real data set and only print out limited information (like the error score every 100 records). So they really serve two different audiences: Hello* for those learning how the algorithms work, and hotgym for those looking for an example they can copy for applying the algorithms to real data.